utils/getInvoices/tests/getInvoices.garbage-collector.test.ts

```diff
@@ -5,142 +5,661 @@ import { expect } from '@jest/globals'
+      it('should NOT load when service = { garbageCollectorPrice: 10 }, company = { discount: 0 }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          discount: 0,
+        }
 
-    expect(invoices).toContainEqual({
-      type: ServiceType.GarbageCollector,
-      price: payment.invoice[0].sum,
-      sum: payment.invoice[0].sum,
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).not.toContainEqual(
+          expect.objectContaining({ type: ServiceType.GarbageCollector })
+        )
+      })
+      it('should load when service = { garbageCollectorPrice: 10 }, company = { rentPart: 10, garbageCollector: true }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 10,
+          garbageCollector: true,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).toContainEqual({ 
+          type: ServiceType.GarbageCollector,
+          price: 1,
+          sum: 1, 
+        })
+      })
+      it('should load when service = { garbageCollectorPrice: 0 }, company = { rentPart: 0, garbageCollector: true }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 0,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 0,
+          garbageCollector: true,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).toContainEqual({ 
+          type: ServiceType.GarbageCollector,
+          price: 0,
+          sum: 0,
+        })
+      })
+      it('should load when service = { garbageCollectorPrice: 10 }, company = { rentPart: 0, garbageCollector: true }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 0,
+          garbageCollector: true,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).toContainEqual({ 
+          type: ServiceType.GarbageCollector,
+          price: 0,
+          sum: 0,
+        })
+      })
+      it('should NOT load when service = { garbageCollectorPrice: 10 }, company = { rentPart: undefined }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: undefined,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).not.toContainEqual(
+          expect.objectContaining({ type: ServiceType.GarbageCollector })
+        )
+      })
```

Comment:
Same as `should NOT load when service = { garbageCollectorPrice: 10 }, company = { ... }`
can be removed as duplicate